orbitale/cms-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "Orbitale\\Bundle\\CmsBundle\\OrbitaleCmsBundle": [
+            "all"
+        ]
```

Comment:
Fixed.
I'm gonna release a 3.0 of this bundle for Symfony3 only + php7 requirements, with some BC breaks (routing config, different dependencies), please not merge until I also have put 3.0 manifest 🙂